packages/subgraph/schema.graphql

```diff
@@ -1,13 +1,297 @@
+  id: ID!
+  "Tenderizer Data"
+  tenderizer: Tenderizer @derivedFrom(field: "protocol")
+  "TenderFarm Data"
+  tenderFarm: TenderFarm @derivedFrom(field: "protocol")
+  "Paritcipating user list"
+  users: [UserProtocol!] @derivedFrom(field: "protocol")
+  "Count of users participated in Protocol"
+  userCount: BigInt!
+}
+
+type Tenderizer @entity {
+  "Name of the protocol eg. 'livepeer'"
+  id: ID!
+  "Protocol entity"
+  protocol: Protocol!
+
+  "Total amount depositied in tenderizer"
+  deposits: BigDecimal!
+  "Total number of tenderizer depostis"
+  depositCount: BigInt!
+  "Total amount withdrawn from tenderizer"
+  withdrawals: BigDecimal!
+  "Total number of tenderizer withdrawals"
+  withdrawalCount: BigInt!
+  "Total rewards collected from protocol"
+  rewards: BigDecimal!
+  "Total rewards collected from protocol in USD"
+  rewardsUSD: BigDecimal!
+  "Total number of times rewards were collected"
+  rewardCount: BigInt!
+  "Total depsotis + total rewards - total withdrawals"
+  currentPrincipal: BigDecimal!
+  "Current Principle in USD"
+  TVL: BigDecimal!
+
+  "Total amount of protocol fees collected from tenderizer"
+  protocolFees: BigDecimal!
+  "Total amount of protocol fees collected from tenderizer in USD"
+  protocolFeesUSD: BigDecimal!
+  "Total amount of liquidity fees collected from tenderizer"
+  liquidityFees: BigDecimal!
+  "Total amount of liquidity fees collected from tenderizer in USD"
+  liquidityFeesUSD: BigDecimal!
+
+  "Day data for tenderizer"
+  dayData: [TenderizerDay!]! @derivedFrom(field: "tenderizer")
+}
+
+type TenderFarm @entity {
+  "Name of the protocol eg. 'livepeer'"
+  id: ID!
+  "Protocol"
+  protocol: Protocol!
+
+  "Total deposits made into farm"
+  deposits: BigDecimal!
+  "Total number of depostits made"
+  depositCount: BigInt!
+  "Total withrawals made from farm"
+  withdrawals: BigDecimal!
+  "Total number of withdrawals made"
+  withdrawalCount: BigInt!
+  "Total rewards collected from farm"
+  harvest: BigDecimal!
+  "Total rewards collected from farm in USD"
+  harvestUSD: BigDecimal!
+  "Total number of harvests made"
+  harvestCount: BigInt!
+  "Total depsotis - total withdrawals"
+  currentPrincipal: BigDecimal!
+  "Total rewards added"
+  rewards: BigDecimal!
+  "Total number of rewards added"
+  rewardCount: BigInt!
+  "CurrentPrincipal in USD"
+  TVL: BigDecimal!
+
+  "Day data for tenderFarm"
+  dayData: [TenderFarmDay!]! @derivedFrom(field: "tenderFarm")
+}
+
+type TenderizerDay @entity {
+  "<Timestamp rounded to current day by dividing by 86400>_<portocol>"
+  id: ID!
+  "Tenderizer"
+  tenderizer: Tenderizer!
+  "The date beginning at 12:00am UTC"
+  date: Int!
+
+  "deposits volume in tenderizer for the day"
+  deposits: BigDecimal!
+  "withdraw volume in tenderizer for the day"
+  withdrawals: BigDecimal!
+  "Total volume for the day: deposits - withdrawals"
+  volume: BigDecimal!
+  "cumulative deposit on that day"
+  cumulativeVolume: BigDecimal!
```

Comment:
This is a useless data point and cumulative values should not be on time series data points